lib/thrift/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ target_compile_definitions(app PUBLIC _GLIBCXX_HAS_GTHREADS=1)
 
 zephyr_library()
 zephyr_include_directories(include)
+zephyr_include_directories(muzic/include)
```

Comment:
Is `muzic` supposed to be available outside this module? (i.e. should you be able to use these headers directly in your app?)

If not, this should use `zephyr_library_include_directories()`.